lib/get-custom-properties.js

```diff
@@ -47,11 +87,28 @@ const rootSelectorRegExp = /^:root$/i;
 const customPropertyRegExp = /^--[A-z][\w-]*$/;
 
 // whether the node is an html or :root rule
-const isHtmlRule = node => node.type === 'rule' && htmlSelectorRegExp.test(node.selector) && Object(node.nodes).length;
-const isRootRule = node => node.type === 'rule' && rootSelectorRegExp.test(node.selector) && Object(node.nodes).length;
+const isHtmlRule = node =>
```

Comment:
I have to presume you have some global prettier configuration or something similar, and that it made all these changes on your behalf. I hope it’s not too much work to unwind them.